backend/picamera_backend.py

```diff
@@ -0,0 +1,98 @@
+from enum import Enum
+from time import time
+from typing import Dict
+
+from backend import Backend, PublishFunc
+from overlay import Canvas
+
+try:
+    from picamera import PiCamera
+    ON_PI = True
+except (ImportError, RuntimeError):
+    ON_PI = False
+
+# Top of window is outside the screen to hide title bar
+PI_WINDOW_TOP_LEFT = (0, -20)
+
+class OverlayLayer(Enum):
```

Comment:
Call this `PiCameraOverlayLayer`? Also, the `OpenCVBackend` technically has layers too. It has base, data and message layers. Should we try to replicate layers in the `OpenCVBackend` as well by creating a similar enum for opencv?